src/azure-cli/azure/cli/command_modules/resource/custom.py

```diff
@@ -2011,7 +2011,7 @@ def list_resource_links(cmd, scope=None, filter_string=None):
 # endregion
 
 
-def rest_call(cmd, method, uri, headers=None, uri_parameters=None,
+def rest_call(cmd, uri, method=None, headers=None, uri_parameters=None,
```

Comment:
1. Why you swap the order?
2. Why you use method='get'?

Reply:
1. Python requires non-default params to appear first
2. This 'get' will be overridden by https://github.com/Azure/azure-cli/blob/f93602026bb1af862eb571ecbd2e469153bba8e4/src/azure-cli/azure/cli/command_modules/resource/_params.py#L304

Reply:
reasonable